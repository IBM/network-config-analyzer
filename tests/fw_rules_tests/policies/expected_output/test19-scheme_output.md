|query|src_ns|src_pods|dst_ns|dst_pods|connection|
|---|---|---|---|---|---|
|connectivity_map, config: np19||||||
||[dlaas,ibm-system-new]|[*]|[kube-system-new-dummy-to-ignore]|[*]|All connections|
||[ibm-system-new,kube-system-new]|[*]|[default]|[*]|All connections|


