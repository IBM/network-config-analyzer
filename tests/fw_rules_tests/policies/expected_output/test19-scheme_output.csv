"query","src_ns","src_pods","src_ip_block","dst_ns","dst_pods","dst_ip_block","connection",
"connectivity_map, config: np19",
"","[dlaas,ibm-system-new]","[*]","","[kube-system-new-dummy-to-ignore]","[*]","","All connections",
"","[ibm-system-new,kube-system-new]","[*]","","[default]","[*]","","All connections",

